common/lua-system.cpp

```diff
@@ -37,7 +42,8 @@ void LuaSystem::On(std::shared_ptr<EntityCreated> data) {
 			script->SetupEnvironment(&this->lua);
 			script->In(comp);
 			LuaScriptMap::Set(entity_id, script);
-		} break;
+		}
+		break;
```

Comment:
Can you put this inside the curly just above it?

Reply:
Yes